msal/src/main/java/com/microsoft/identity/client/SingleAccountPublicClientApplication.java

```diff
@@ -95,4 +119,65 @@ public void onError(Exception exception) {
                     }
                 });
     }
+
+    /**
+     * Get current account that is persisted in shared preference.
+     *
+     * @return a persisted MultiTenantAccount. This could be null.
+     */
+    @Nullable
+    private MultiTenantAccount getPersistedCurrentAccount() {
```

Comment:
Ideally we might separate this functionality into another delegate class for `SingleAccountPublicClientApplication`, I'm fine to see this change in a follow-up PR -- persisting/restoring doesn't seem like it should be the direct responsibility of this class.